proj3_chess/chess_pieces.py

```diff
@@ -0,0 +1,130 @@
+from board import Board
+from player import Player
+
+
+class ChessPiece:
+    """Mother class for all chess pieces."""
+
+    piece_name = ""
```

Comment:
This is a constant. Yes, it will be overridden in child classes, but that doesn't mean it's value has changed.